feat(store): add xbox series x and s

vnguyen00 opened the pull request:
<!-- Please use Conventional Commits to label your title -->
<!-- https://www.conventionalcommits.org/en/v1.0.0/ -->
<!-- Example: feat: allow provided config object to extend other configs  -->

### Description

Closes #506

Adding Xbox Series X and S to the scanner
<!-- Fixes #(issue) -->
<!-- Please also include relevant motivation and context. -->
I am trying to purchase an Xbox on release day :)

### Testing
Ran npm start run locally and scanner passed back expected results.
<!-- Please describe the tests that you ran to verify your changes. -->
went through lint
<!-- Provide instructions so we can reproduce. -->
npm start run
<!-- Please also list any relevant details for your test configuration -->
modified models in .env file to search for xboxsx and xboxss

vnguyen00 commented:
The Walmart XBX page will error out for now. The page is non-functional for now. It should keep the same URI
[11:20:54 AM] warn :: ✖ [walmart] [microsoft (xboxsx)] xbox series x :: STATUS CODE ERROR 404

jef commented:
Overall, looks good to me!

> The Walmart XBX page will error out for now. The page is non-functional for now. It should keep the same URI
> [11:20:54 AM] warn :: ✖ [walmart] [microsoft (xboxsx)] xbox series x :: STATUS CODE ERROR 404

I don't mind this error for now since it'll be stated; but how did you end up getting these IDs?

vnguyen00 commented:
> I don't mind this error for now since it'll be stated; but how did you end up getting these IDs?

From this page, https://www.walmart.com/cp/xbox-series/9206773 .
The series S link works just not the X.

mven commented:
Don't forget about the additions to the readme.

v1ct0rv commented on src/store/model/amazon.ts:
```diff
@@ -273,6 +273,18 @@ export const Amazon: Store = {
+			brand: 'microsoft',
+			model: 'xbox series x',
+			series: 'xboxsx',
+			url: 'https://www.amazon.com/Xbox-X/dp/B08H75RTZ8/ref=sr_1_1?crid=2LDCJ421GASAD&dchild=1&keywords=xbox+series+x&qid=1604939295&sprefix=xbox+seri%2Caps%2C245&sr=8-1'
```
```suggestion
			url: 'https://www.amazon.com/dp/B08H75RTZ8'
```

v1ct0rv commented on src/store/model/amazon.ts:
```diff
@@ -273,6 +273,18 @@ export const Amazon: Store = {
+			brand: 'microsoft',
+			model: 'xbox series s',
+			series: 'xboxss',
+			url: 'https://www.amazon.com/Xbox-X/dp/B08G9J44ZN/ref=sr_1_1?crid=2LDCJ421GASAD&dchild=1&keywords=xbox%2Bseries%2Bx&qid=1604939295&sprefix=xbox%2Bseri%2Caps%2C245&sr=8-1&th=1'
```
```suggestion
			url: 'https://www.amazon.com/dp/B08G9J44ZN'
```

v1ct0rv commented:
> Don't forget about the additions to the readme.

Yes, the README is very important.

vnguyen00 commented:
> Don't forget about the additions to the readme.

Done. I added to supported 

> > Don't forget about the additions to the readme.
> 
> Yes, the README is very important.

Done! Added to the brands and models section. LMK if there's anything else.

jef reviewed (approved):
Thanks!
